lib/fluent/plugin/s3_compressor_arrow.rb

```diff
@@ -26,7 +32,11 @@ def configure(conf)
           raise Fluent::ConfigError, "#{@arrow.format} unsupported with #{@arrow.format}"
         end
 
-        @schema = Arrow::Schema.new(@arrow.schema)
+        if !(@arrow.schema.nil? ^ @arrow.schema_from.nil?)
```

Comment:
```suggestion
        if @arrow.schema.nil? and @arrow.schema_from.nil?
```

Reply:
I want to make it a condition of `not xor`. I would like to accept the following settings.
|  schema  |  schema_from  |  result  |
| ---- | ---- | ---- |
|  T  |  T  |  invalid  |
|  T  |  F  |  valid  |
|  F  |  T  |  valid  |
|  F  |  F  |  invalid  |

Reply:
Sorry!